wp-job-manager.php

```diff
@@ -66,6 +71,9 @@ public function __construct() {
 		include( 'includes/class-wp-job-manager-forms.php' );
 		include( 'includes/class-wp-job-manager-geocode.php' );
 		include( 'includes/class-wp-job-manager-cache-helper.php' );
+		include( 'includes/rest-api/class-wp-job-manager-rest-api.php' );
+
+		add_action( 'init', array( $this, 'rest_api' ) );
```

Comment:
Good point! We can, I'll have to change some things around on MT, definitely doable and sensible.